examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -48,6 +48,8 @@
         "Test_TC_PSCFG_2_1",
         "Test_TC_PSCFG_2_2",
         "Disabled because darwin-framework-tool does not support GetCommissionerRootCertificate",
-        "Test_AddNewFabricFromExistingFabric"
+        "Test_AddNewFabricFromExistingFabric",
+        "Disabled in darwin due to comment: https://github.com/project-chip/connectedhomeip/pull/28985#issuecomment-1701574449",
```

Comment:
Please add details beyond a link, otherwise we are forced to open github when reading comments which is very inconvenient.

Adding some more details like:

```
"Darwin C++ build fails with EMBER_ZCL_STATUS_INVALID_IN_STATE undefined (suggests EMBER_ZCL_STATE_INVALID_IN_STATE as being defined"
```

Also given the above ... is the error correct? should we name things differently. @bzbarsky-apple or @vivien-apple for codegen

Reply:
Grepping existing tests, it seems like existing errors suported are:

```
          error: BUSY
          error: CONSTRAINT_ERROR
          error: FAILSAFE_REQUIRED
          error: FAILURE
          error: INVALID_ACTION
          error: INVALID_COMMAND
          error: INVALID_DATA_TYPE
          error: NEEDS_TIMED_INTERACTION
          error: NOT_FOUND
          error: RESOURCE_EXHAUSTED
          error: SUCCESS
          error: UNSUPPORTED_ACCESS
          error: UNSUPPORTED_ATTRIBUTE
          error: UNSUPPORTED_CLUSTER
          error: UNSUPPORTED_COMMAND
          error: UNSUPPORTED_ENDPOINT
          error: UNSUPPORTED_WRITE
 ```
 
 and all of these seem to have a value in `src/app/util/af-enums.h`

Reply:
#28530 apparently named it state instead of status

Reply:
created #29013 to fix this

Reply:
Updated please verify.